### PR TITLE
Add guidelines to nominate new committers

### PR DESCRIPTION
**Pull request by tgraf**
Establishes guidelines to add new committers to the project. The guidelines are based on the respective guidelines of the Open vSwitch process as they are very well written have proven to be effective to recognize contributions to the open source project.

### CODE REVIEW

**Comment by tgraf** [comment]
@borkmann @aanm @mchalla Since all of you are current committers, please approve the suggested guidelines after careful review.

Of course, everyone else is also welcome to review this and suggest changes.

**Comment by borkmann** [comment]
The guidelines look good to me, approved:

Signed-off-by: Daniel Borkmann <daniel@cilium.io>

**Comment by aanm** [comment]
LGTM

**Review by mchalla — APPROVED** [review]
LGTM

**Pull request merged**
